packages/gatsby-plugin-nginx/src/nginx-generator.ts

```diff
@@ -16,6 +17,15 @@ export interface NginxDirective {
   children?: NginxDirective[]
 }
 
+export type NginxRewriteType = 'proxy' | 'rewrite' | 'redirect' | 'error_page'
```

Comment:
Can you move these type definitions to `typings/types.d.ts`? Also in my opinion you can extend the `Redirect` type defined there instead of introducing `NginxRedirect`

Reply:
I use `NginxDirective` here, should I move it too to `typings/types.d.ts`?

Reply:
Good catch. Sure, move it as well

Reply:
Done!